Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -39,18 +45,22 @@
 <summary>Examples</summary>
 
 - I was consistently in demand for design and code review.
-- I was recognized by engineers in multiple teams as an expert mentor and teacher.
+- I am actively mentoring multiple Octopus engineers.
+- Engineers at Octopus seek out to work on my teams to learn from me.
+- After reviewing a code contribution from an engineer I saw could be improved, I worked directly with that engineer and their manager to uplift their skills in the identified growth area.
 - I helped a team make a course correction based on new information or ideas.
 - I provided appropriate technical freedom, and framing, that allowed the Lead Software Engineers on my teams to thrive independently.
 
 </details>
 
 ## 🧭 Culture and Leadership
 
+- I model our cultural safety by thinking in the open, asking imperfect questions, being wrong, and accepting corrections.
```

Comment:
Terms like "model our cultural safety" and "imperfect questions" felt a bit abstract/open to interpretation. Here's a reword to consider, which puts the intended _outcome_ upfront:

```suggestion
- I create an environment where others feel safe, by modeling vulnerability, thinking in the open, being wrong, and accepting corrections.
```